doc/source/tune-config.rst

```diff
@@ -69,6 +69,30 @@ By default, each random variable and grid search point is sampled once. To take
 
 For more information on variant generation, see `variant_generator.py <https://github.com/ray-project/ray/blob/master/python/ray/tune/variant_generator.py>`__.
 
+Broadcasting Objects
```

Comment:
`Handling Large Datasets/Broadcasting Objects`? (more relevance to supervised learning users)

Reply:
Updated